taskrun/VerboseObserver.py

```diff
@@ -213,6 +197,35 @@ def task_failed(self, task, errors):
 
     self._progress()
 
+  def task_killed(self, task):
+    """
+    See Observer.task_killed()
+    """
+
+    if self._timer:
+      task_time = time.time() - self._times.pop(task)
+
+    self._finished_tasks += 1
+    self._killed_tasks += 1
+    if self._show_kills:
+      # format the output string
+      text = '[Killed: {0}'.format(task.name)
+      # optionally add the time
+      if self._timer:
+        text += ' {0}'.format(_time_string(task_time))
+      text += ']'
+      # add the description
+      text += '\n  Description: {0}'.format(task.describe())
```

Comment:
this should be wrapped in if self._show_description, this shouldn't clog up the output otherwise

Reply:
The comment in the constructor says that descriptions are always shown for "errors", which now means failures and killings. Thoughts?

Reply:
Change that to say descriptions are always shown for "failures"?